src/tests/integration_test_utils.rs

```diff
@@ -185,32 +212,51 @@ impl DeltaTestOutput {
     #[allow(unused)]
     pub fn inspect_raw(self) -> Self {
         eprintln!("{}", "▼".repeat(100));
-        eprintln!("{}", self.output);
+        eprintln!("{}", self.raw_output);
         eprintln!("{}", "▲".repeat(100));
         self
     }
 
-    pub fn explain_ansi(mut self) -> Self {
-        self.explain_ansi_ = true;
+    pub fn expect_skip(self, skip: usize, expected: &str) -> Self {
+        lines_match(expected, &self.output, Some(skip));
         self
     }
 
-    pub fn expect_skip(self, skip: usize, expected: &str) -> String {
-        let processed = self.format_output();
-        lines_match(expected, &processed, Some(skip));
-        processed
+    pub fn expect(self, expected: &str) -> Self {
+        self.expect_skip(crate::config::HEADER_LEN, expected)
     }
 
-    pub fn expect(self, expected: &str) -> String {
-        self.expect_skip(crate::config::HEADER_LEN, expected)
+    pub fn expect_contains(self, expected: &str) -> Self {
+        let output_contains_text = self.output.contains(expected);
+        if !output_contains_text {
+            println!("Output does not contain \"{}\":\n{}", expected, self.output);
+            assert!(output_contains_text);
```

Comment:
I'm thinking that the printed message should go to stderr; the `assert!` macro actually supports that -- the [optional second argument](https://doc.rust-lang.org/std/macro.assert.html#custom-messages) is a string message which will be written to stderr.

Reply:
Nice! That simplifies things.